django_app/redbox_app/redbox_core/migrations/0003_file.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.11 on 2024-03-20 08:18
+
+from django.db import migrations, models
```

Comment:
note to self, auto linting doesnt appear to be working..... this is a problem for me to fix